forch/forchestrator.py

```diff
@@ -484,7 +484,7 @@ def _register_handlers(self):
 
     def _get_varz_config(self):
         metrics = self._varz_collector.retry_get_metrics(
-            self._faucet_prom_endpoint, _TARGET_FAUCET_METRICS)
+                self._faucet_prom_endpoint, _TARGET_FAUCET_METRICS)
```

Comment:
is this right?  I thought the hanging indent was supposed to be 4... ?

Reply:
Yeah, that's an oversight. Correcting.